src/net/server/channel/handlers/DueyHandler.java

```diff
@@ -329,11 +340,11 @@ private void addItemToDB(Item item, int quantity, int mesos, String sName, int r
     private String getCurrentDate() {
         String date = "";
         Calendar cal = Calendar.getInstance();
-        int day = cal.get(Calendar.DATE) - 1; // instant duey ?
+        int day = cal.get(Calendar.DATE) + (ServerConstants.USE_DUEY_INSTANT_DELIVERY ? -1 : 0); // instant duey ?
```

Comment:
why do we add the conditional here `+ (ServerConstants.USE_DUEY_INSTANT_DELIVERY ? -1 : 0)`

Reply:
If we set the timestamp to the current day, the package will be shown as still delivering, presumably until the next day. Having this line allows for instant package claiming.

Though I also realized a bug that can occur on the first day of each month.

Reply:
It seems like having 00 as the day doesn't cause an error.